recipe/bld.bat

```diff
@@ -1,11 +1,9 @@
-:: There is no `gdal-config` on Windows so we need to hardcode gdal's version.
-if "%libgdal%" == "2.4" (
-  echo "set GDAL_VERSION=2.4.3"
-  set GDAL_VERSION=2.4.3
-) else (
-  echo "set GDAL_VERSION=3.0.2"
-  set GDAL_VERSION=3.0.2
+:: There is no `gdal-config` on Windows so we need figure it out from gdalinfo
+for /F "USEBACKQ tokens=2 delims=, " %%F in (`gdalinfo --version`) do (
+set GDAL_VERSION=%%F
 )
+if errorlevel 1 exit 1
+echo "set GDAL_VERSION=%GDAL_VERSION%"
```

Comment:
@ocefpaf, does this look (vaguely) like what you had in mind?

Reply:
Perfect! Thanks!!

Reply:
It looks like we're good!  Honestly, it was helpful to remind myself how to parse out version numbers from conda, even if it doesn't get used here.